### PR TITLE
Lobby UI Visual Rework

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description (*)

A bit of a facelift for the lobby UI--mainly moving things around, adding color to the scroll bar, adding some graphics, and adding borders to the tabs to make them more like radio buttons

New Look:
image
image
image
image
image

Old Look:
image
image
image
image
image

### Issue Number(s) (*)

Jira ticket [here](https://jira.unity3d.com/browse/MTT-2735)

### CODE REVIEW

**Review by SamuelBellomo — DISMISSED** [review]

**Review by LPLafontaineB — APPROVED** [review]

**Review by fernando-cortez — COMMENTED** [review]
Will approve once merge conflict is addressed. Looks much better👍

**Review by LPLafontaineB — APPROVED** [review]

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**
